docs/running-on-yarn.md

```diff
@@ -495,6 +468,20 @@ To use a custom metrics.properties for the application master and executors, upd
   name matches both the include and the exclude pattern, this file will be excluded eventually.
   </td>
 </tr>
+<tr>
+  <td><code>spark.yarn.report.interval</code></td>
+  <td>1s</td>
+  <td>
+  Interval between reports of the current app status in YARN cluster mode.
+  </td>
+</tr>
+<tr>
+  <td><code>spark.yarn.services</code></td>
```

Comment:
I don't think we should expose this.  I think this was added with a PR to start adding integration with ATS which I don't know we officially support yet, this config is just for testing at this point.  @steveloughran to verify.